File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/factories/BandaFactoryTest.java
package simulacionBandaMusica.factories;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.models.Banda;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class BandaFactoryTest {

    @Test
    void createBandaTest() {
        Banda banda = BandaFactory.create();
        // Banda no nulo, para ver si se han asignado correctamente los datos
        for (int i = 0; i < banda.vectorMusicos().length; i++) {
            assertNotNull(banda.vectorMusicos()[i]);
        }
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/factories/BandaFactoryTest.kt
package simulacionBandaMusica.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class BandaFactoryTest {
    @Test
    fun createBandaTest() {
        val banda = BandaFactory.create()
        // Banda no nulo, para ver si se han asignado correctamente los datos
        for (i in 0 until banda.vectorMusicos.size) {
            Assertions.assertNotNull(banda.vectorMusicos[i])
        }
    }
}